docs/volume_annotation.md

```diff
@@ -28,16 +28,16 @@ In the `Segmentation` tab on the right-hand side, you can see the cell IDs which
 
 ### Proof-Reading and Merging Segments
 
-webKnossos support proof-reading of segments from automatic segmentations. With "Merger Mode" individual segments (e.g. from over-segmentation) can be combined to refine the segmentation. 
+webKnossos support proof-reading of segments from automatic segmentation. With "Merger Mode" individual segments (e.g. from over-segmentation) can be combined ("merged") to refine the segmentation and fix split errors. 
 
-The "merger mode" is available in skeleton and hybrid annotation mode. Mark connected segments by right clicking and placing nodes in the corresponding segments to merge them together. Several segments can be combined by making sure that all "correcting nodes" are part of the same tree.
+The "merger mode" is available in skeleton and hybrid annotation mode. Mark connected -segments by right-clicking and placing nodes in the corresponding segments to merge them together. Several segments can be combined by making sure that all "correcting nodes" are part of the same tree.
```

Comment:
>connected -segments

I think there is a little mistake here. Do you mean `connected-segments` or `connected segments`?